common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -18,5 +18,33 @@
   "feature.wynntils.itemHighlight.name": "Item Highlights Feature",
   "featureDebug.wynntils.connectionProgress.name": "Connection Progress Feature",
   "featureDebug.wynntils.keyBindTest.name": "Connection Progress Feature",
-  "featureDebug.wynntils.packetDebugger.name": "Packet Debugger Feature"
+  "featureDebug.wynntils.packetDebugger.name": "Packet Debugger Feature",
+  "feature.wynntils.lootrunUtils.recordStart":"Started recording lootrun. Execute %s to stop recording.",
```

Comment:
shouldn't these go above the debug strings?

i feel like it would be good to have some convention for organizing stuff in this file, too, like alphabetical order. can we configure the formatter to do that automatically?

Reply:
Alphabetic order sounds good. I don't think we can get this just by setting a switch, but we can probably hook in our own formatter to do this.

Reply:
#58